app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass

```

Comment:
It's a good practice to provide a default message for your custom exceptions. This way, if an error occurs and no specific message is provided, your exception will still give some information about what went wrong. You can do this by overriding the `__init__` method in your exception class. For example: `class VaccineError(Exception): def __init__(self, message='Default VaccineError message'): super().__init__(message)`